.github/actions/services-validator/action.yaml

```diff
@@ -46,8 +52,8 @@ runs:
           eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
           echo "/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin" >> $GITHUB_PATH
         fi
-        brew install --overwrite --quiet python3
-        python3 -m pip install jsonschema json_source_map requests
+        brew install --overwrite --quiet python@3.11
+        python3 -m pip install jsonschema json_source_map requests aiohttp aiodns
```

Comment:
`aiodns` doesn't seem to be used anywhere.

Reply:
aiohttp will use it automatically to speed up lookups, probably overkill though since we only have to do a few.

Reply:
Ah interesting, forgot about implicit library optimizations